lua/soulbound_abilities/beebarrel.lua

```diff
@@ -0,0 +1,137 @@
+local ABILITY = {}
+
+ABILITY.Name = "Place Bee Barrel"
+ABILITY.Id = "beebarrel"
+ABILITY.Description = "Place a bee barrel that will release bees when it explodes"
+ABILITY.Icon = "vgui/ttt/roles/sbd/abilities/icon_beebarrel.png"
+
+local beebarrel_uses = CreateConVar("ttt_soulbound_beebarrel_uses", "3", FCVAR_REPLICATED, "How many uses of the place bee barrel ability should the Soulbound get. (Set to 0 for unlimited uses)", 0, 10)
+local beebarrel_bees = CreateConVar("ttt_soulbound_beebarrel_bees", "3", FCVAR_REPLICATED, "How many bees per beebarrel", 1, 10)
```

Comment:
This needs to be added to README.md and the convars table

Reply:
I added it to the README.md, but where is the convars table?

Reply:
Line 12 below

Reply:
I did it, thanks